tests/test_quoteengine.py

```diff
@@ -0,0 +1,124 @@
+from datetime import datetime
+from unittest.mock import patch
+from unittest.mock import MagicMock
+
+import pytest
+
+from tululbot.utils import QuoteEngine
+
+
+class FakeResponse:
+    def get_json(self):
+        return self.json

```

Comment:
what's the value of `self.json`? you don't create a constructor for this class so I suspect this must be `None`. is this expected?